db/database.js

```diff
@@ -1,37 +1,66 @@
 const mongoose = require('mongoose');
+const sampleData = require('../data/sampleData.js');
 
 mongoose.connect('mongodb://localhost/fetcher');
 
+const db = mongoose.connect;
 
-// stubbing out this actual schema for my real data
-// let repoSchema = mongoose.Schema({
-//   id: {
-//     type: String,
-//     unique: true,
-//   },
-//   restaurant: String,
-//   about: {
-//     description: String,
-//     hours: String,
-//     cost: String,
-//     style: String,
-//     phone: Number,
-//   },
-//   banner: String,
-//   photo: [],
-// });
-
-
-const testSchema = mongoose.Schema({
-  id: Number,
-  text: String,
+
+const aboutSchema = mongoose.Schema({
+  id: {
+    type: Number,
+    unique: true,
+  },
+  restaurant: String,
+  about: {
+    description: String,
+    hours: String,
+    cost: String,
+    style: String,
+    phone: String,
+  },
+  banner: String,
+  photo: [],
 });
 
 
-const Repo = mongoose.model('Repo', testSchema);
+const About = mongoose.model('About', aboutSchema);
+
+let count = 0;
+
+sampleData.forEach((data) => {
+  const obj = { id: '', about: { } };
+  obj.id = data.id;
+  obj.restaurant = data.name;
+  obj.about.description = data.about.description;
+  obj.about.hours = data.about.hours;
+  obj.about.cost = data.about.price;
```

Comment:
I think you can just do
```
data.about.cost = data.about.price;
const about = new About(data);
```
instead of re-creating obj directly from data (since they are so similar).  I believe unused properties are just discarded by Mongoose.